packages/react/src/Timeline/__tests__/Timeline.test.tsx

```diff
@@ -49,4 +88,80 @@ describe('Timeline.Badge', () => {
     const results = await axe.run(container)
     expect(results).toHaveNoViolations()
   })
+
+  it('should support `className` on the outermost element', () => {
+    const Element = () => <Timeline.Badge className={'test-class-name'} />
+    const FeatureFlagElement = () => {
+      return (
+        <FeatureFlags
+          flags={{
+            primer_react_css_modules_team: true,
+            primer_react_css_modules_staff: true,
+            primer_react_css_modules_ga: true,
+          }}
+        >
+          <Element />
+        </FeatureFlags>
+      )
+    }
+    expect(HTMLRender(<FeatureFlagElement />).container.firstChild?.firstChild).toHaveClass('test-class-name')
+  })
+})
```

Comment:
The sx-compliant Timeline.Badge doesn't support className, I thought it'd make sense to add it to the css modules refactor and therefore I'm only testing for that one here, but lmk if that doesn't make sense and I can revert!